candid/evm_rpc.did

```diff
@@ -139,6 +139,7 @@ type Provider = record {
   chainId : ChainId;
   urlPattern : text;
   headerPatterns : opt vec HttpHeader;
+  "service" : opt RpcService;
```

Comment:
nit: are the quotes needed because "service" is a reserved keyword in Candid? Maybe renaming it to `rpc_service` would be easier. WDYT?

Reply:
> are the quotes needed because "service" is a reserved keyword in Candid?

That's correct. We could potentially also rename this to `alias`; do you think that's clear enough in your opinion?

Reply:
great idea, I like `alias`!